docs/discovery/bos/components.md

```diff
@@ -0,0 +1,132 @@
+---
+sidebar_position: 3
+---
+
```

Comment:
This file looks like is not needed, since we have this information across other links.